demos/checkbox.html

```diff
@@ -163,27 +190,141 @@
       </section>
+
+      <section class="example section-dark-theme">
+        <h2>Dark Theme (mdc-checkbox--theme-dark) - CSS Only</h2>
+        <div>
+          <div class="mdc-form-field mdc-form-field--theme-dark">
+            <div id="mdc-checkbox-dark" class="mdc-checkbox mdc-checkbox--theme-dark">
+              <input type="checkbox"
+                     id="checkbox-dark"
+                     class="mdc-checkbox__native-control"/>
+              <div class="mdc-checkbox__background">
+                <svg class="mdc-checkbox__checkmark"
+                     viewBox="0 0 24 24">
+                  <path class="mdc-checkbox__checkmark__path"
+                        fill="none"
+                        stroke="white"
+                        d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
+                </svg>
+                <div class="mdc-checkbox__mixedmark"></div>
+              </div>
+            </div>
+            <label for="checkbox-dark" id="dark-checkbox-label">This is my checkbox</label>
+          </div>
+        </div>
+        <div>
+          <div class="mdc-form-field mdc-form-field--theme-dark">
+            <div class="mdc-checkbox mdc-checkbox--theme-dark mdc-checkbox--disabled">
+              <input type="checkbox"
+                     id="basic-disabled-checkbox"
+                     class="mdc-checkbox__native-control"
+                     disabled />
+              <div class="mdc-checkbox__background">
+                <svg class="mdc-checkbox__checkmark"
+                     viewBox="0 0 24 24">
+                  <path class="mdc-checkbox__checkmark__path"
+                        fill="none"
+                        stroke="white"
+                        d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
+                </svg>
+                <div class="mdc-checkbox__mixedmark"></div>
+              </div>
+            </div>
+            <label for="basic-disabled-checkbox" id="basic-disabled-checkbox-label">This is my disabled checkbox</label>
+          </div>
+        </div>
+        <div>
+          <div class="mdc-form-field mdc-form-field--theme-dark">
+            <div class="mdc-checkbox mdc-checkbox--theme-dark">
+              <input type="checkbox"
+                     id="basic-indeterminate-checkbox-dark-2"
+                     class="mdc-checkbox__native-control"/>
+              <div class="mdc-checkbox__background">
+                <svg class="mdc-checkbox__checkmark"
+                     viewBox="0 0 24 24">
+                  <path class="mdc-checkbox__checkmark__path"
+                        fill="none"
+                        stroke="white"
+                        d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
+                </svg>
+                <div class="mdc-checkbox__mixedmark"></div>
+              </div>
+            </div>
+            <label for="basic-indeterminate-checkbox-dark" id="basic-disabled-checkbox-label">This is my indeterminate checkbox</label>
```

Comment:
`<label for="basic-indeterminate-checkbox-dark-2" id="basic-indeterminate-checkbox-dark-2-label">`